app/routes/models/delete.js

```diff
@@ -0,0 +1,33 @@
+function ViewModel (value, error) {
+  // Constructor function to create logic dependent nunjucks page
+  this.model = {
+    id: 'delete',
+    name: 'delete',
+    fieldset: {
+      legend: {
+        text: 'Are you sure you want to delete your application?',
+        isPageHeading: true,
+        classes: 'govuk-fieldset__legend--l'
+      }
+    },
+    items: [
+      {
+        value: true,
+        text: 'Yes'
+      },
+      {
+        value: false,
+        text: 'No'
+      }
+    ]
+  }
+
+  // If error is passed to model then this error property is added to the model
+  if (error) {
+    this.model.errorMessage = {
+      text: 'Please select if you would like to delete your application.'
```

Comment:
Maybe change to:

'Please select if you would like to delete your agreement.'

Reply:
Done